Header header
string car_id #检测小车的id
nav_msgs/Odometry odom
Detection[] detections #目标检测结果